### PR TITLE
docs(edits): overview text

### PR DESCRIPTION
**Pull request by heathkat**
edit the Overview text and add icon button to clear search box

### CODE REVIEW

**Comment by corykon** [comment]
Could use the new [search bar component](https://cashmere.healthcatalyst.net/web/components/search-bar/examples) here?

searchbar

**Comment by heathkat** [comment]
@corykon that's a great idea! I'm going to try that tomorrow and add to this PR.

**Review by andrew-frueh — APPROVED** [review]
Exciting!  First official use of @corykon's `hc-search-bar` :-)

File: src/app/content/usage/usage-list/usage-list.component.html [hunk 1]
```diff
@@ -2,21 +2,17 @@
     <hc-tab tabTitle="Term List" tabindex="2">
         <div>
             <div class="usage-table-filters">
-                <hc-form-field class="usage-table-filters-input" inline>
-                    <hc-label>Filter:</hc-label>
-                    <input hcInput (keyup)="applyFilter()" [formControl]="searchControl" placeholder="Search" />
-                    <hc-icon hcSuffix fontSet="fa" fontIcon="fa-times" (click)="resetSearch()"></hc-icon>
-                </hc-form-field>
+                <hc-search-bar [placeholder]="placeholder" [showSearchIcon]="showSearchIcon" [showClearIcon]="showClearIcon" (triggerSearch)="applyFilter($event)"></hc-search-bar>
```
Comment (corykon) [inline src/app/content/usage/usage-list/usage-list.component.html:5]: You could actually simplify this even further. THe example on cashmere is illustrating all of the inputs, but most of them default to what you are setting them to. You could remove all the properties you added in typescript and just have this in your template:

```
<hc-search-bar (triggerSearch)="applyFilter($event)"></hc-search-bar>
```

File: src/app/content/usage/usage-list/usage-list.component.ts [hunk 2]
```diff
@@ -16,14 +17,22 @@ import {IUsage, usageAttributesMapping} from '../usage';
...
+    categories = ['All', 'Clinical', 'General', 'Health Catalyst', 'Industry', 'Life sciences', 'Technical'];
+    types = ['All', 'Abbreviation', 'General usage', 'UX/technical writing', 'Word choice'];
     selectedCategoriesControl = new FormControl('All');
     selectedTypesControl = new FormControl('All');
     searchControl = new FormControl();
+    searchTerm = '';
     termList$: Observable<IUsage[]>;
     terms: IUsage[];
 
+    placeholder = 'Search';
```
Comment (corykon) [inline src/app/content/usage/usage-list/usage-list.component.ts:29]: Won't need lines 29-34 at all :)

**Review by corykon — APPROVED** [review]

File: src/app/content/usage/usage-list/usage-list.component.html (hunk 1 above)
Comment (heathkat) [reply src/app/content/usage/usage-list/usage-list.component.html:5]: Oh cool! Thanks for pointing that out @corykon I will give it a try!

**Pull request merged**